certified-connectors/Tendocs Documents/apiDefinition.swagger.json

```diff
@@ -108,9 +108,243 @@
             "description": "Something has gone wrong, please contact support@tendocs.com"
           }
         },
-        "summary": "Convert Document To New File Format",
+        "summary": "Convert Document To New File Format (PDF/HTML/TXT)",
         "description": "Convert a document to a new file format. Includes PDF and HTML."
       }
+    },
+    "/ai/v1/tasks/summary": {
+      "post": {
+        "tags": [
+          "Documents",
+          "AI",
+          "Summary"
+        ],
+        "operationId": "V1AiSummaryPost",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AiSummaryRequest"
+            }
```

Comment:
Kindly resolve below validation error

##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1ai~1v1~1tasks~1summary/post/responses/default